<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>input_qua</class>
 <widget class="QWidget" name="input_qua">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>input_qua</string>
  </property>
  <widget class="QPushButton" name="yes">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="name_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ordx">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1499.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ordy">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>130</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <double>950.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ordx_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1499.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ordy_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <double>950.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="title2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>91</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; font-style:italic;&quot;&gt;X&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="title2_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; font-style:italic;&quot;&gt;X&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="title3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; font-style:italic;&quot;&gt;Y&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="title3_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>270</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; font-style:italic;&quot;&gt;Y&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="title1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;地点名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="title1_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;地点名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>yes</sender>
   <signal>clicked()</signal>
   <receiver>input_qua</receiver>
   <slot>on_re__clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_re__clicked()</slot>
 </slots>
</ui>
